src/Resources/public/Admin.js

```diff
@@ -154,6 +155,10 @@ var Admin = {
                   $(e.target).trigger('change');
               });
 
+            // See https://github.com/sonata-project/SonataAdminBundle/pull/6466
```

Comment:
Not sure it's necessary to add a link for the PR introducing this code, it's already in the commit.